tests/Bundle/CoverallsBundle/Collector/CiEnvVarsCollectorTest.php

```diff
@@ -232,6 +263,23 @@ public function shouldHaveReadEnvAfterCollectTravisCiEnvVars(CiEnvVarsCollector
         $this->assertArrayHasKey('CI_NAME', $readEnv);
     }
 
+    /**
+     * @test
+     * @depends shouldCollectGithubActionsEnvVars
+     *
+     * @param CiEnvVarsCollector $object
+     */
+    public function shouldHaveReadEnvAfterCollectGithubActionsEnvVars(CiEnvVarsCollector $object)
```

Comment:
I wonder how is it possible to use param type hint on PHP 5.5? 🤔Build is green

Reply:
Looks like that build runs only php 7.1.2 in appveyor

Reply:
Tests are run in 5.5 https://travis-ci.org/php-coveralls/php-coveralls/jobs/632963135 as well. Mysterious :D

Reply:
@keradus, can we delete ci for php 5.5?